### PR TITLE
Use .ruby-version to control ruby version in local dev and ci

### PR DESCRIPTION
**Pull request by ecwall**

### CODE REVIEW

File: build/teamcity-test.sh [hunk 1]
```diff
@@ -16,9 +24,6 @@ run_cockroach() {
   # Start a CockroachDB server, wait for it to become ready, and arrange
   # for it to be force-killed when the script exits.
   rm -f "$urlfile"
-  # Clean out a past CockroachDB instance. This will clean out leftovers
-  # from the build agent, and also between CockroachDB runs.
-  cockroach quit --insecure || true
```
Comment (ecwall) [inline build/teamcity-test.sh:21]: `cockroach quit` no longer exists

File: .ruby-version [hunk 2]
```diff
@@ -0,0 +1 @@
+2.7.7
```
Comment (rafiss) [inline .ruby-version:1]: is this actually what this project was using before? i wonder if we can get it onto 3.1.3

File: build/teamcity-test.sh (hunk 1 above)
Comment (rafiss) [reply build/teamcity-test.sh:21]: do we need something else that shuts down the process?

**Review by rafiss — APPROVED** [review]

**Comment by ecwall** [comment]
CI tests are failing, but they fail without this change also

**Pull request merged**